### DISCUSSION TITLE
费用问题

### DISCUSSION THREAD

**Discussion by maijializi** [illacloud/illa-builder#1584] [中文支持 Chinese Support]
illa是怎么收费的

**Comment by donqu1xotechina** [comment]
hello 您可以参考我们的pricing页面：https://www.illacloud.com/pricing 。 目前开源的self hos版本是完全免费的。

**Reply by lucktoo** [reply]
请问差别在那？开源与收费的？

**Reply by donqu1xotechina** [reply]
Hey：
开源的需要自己部署，动手能力强的用户更好一些。收费的可以一键部署，整体更方便快捷。
